public/assets/tasks.json

```diff
@@ -0,0 +1,44 @@
+[
+  {
```

Comment:
Instead of using this json file, you could import this json file into the mongo database and fetch data from the database